### ISSUE TITLE
Add a column for number of pull requests made in participants list

### ISSUE THREAD

**Issue by akashlakhera** [CodeDocsJECRC/hacktoberfest2018#27]
The URL "hacktoberfest.digitalocean.com/stats/username" contains the number of PR's done by the user. We use the data from the site to dynamically update the PR column in participants list. But I'm not sure if Github Markdown supports dynamic text.

**Comment by Man-Jain** [comment]
Yes we can do this but till then we should just give the link in the README file.

**Comment by maddhruv** [comment]
check #48

**Comment by vipulgupta2048** [comment]
For now, #48 is a good hack. I am more interested in ways we can dynamically update no. of PR's stored in a table of markdown format on GitHub. If I understand the issue correctly that is.
